search.py

```diff
@@ -0,0 +1,93 @@
+"""Functions for searching"""
+import sys
+from datetime import datetime, timedelta
+
+import requests
+from requests.exceptions import ConnectionError
+
+
+def get_url(code_from, code_to, dep_date, ret_date, trip_type='TT'):
+    """a method that calculates url"""
+    payload = [(trip_type, 'RT'), ('SS', ''), ('RT', ''), ('FL', 'on'), ('DC', code_from),
+               ('AC', code_to), ('AM', dep_date[0:7]), ('AD', dep_date[8:]),
+               ('DC', ''), ('AC', ''), ('AM', ''), ('AD', ''), ('DC', ''), ('AC', ''),
+               ('AM', ''), ('AD', ''), ('DC', ''), ('AC', ''), ('AM', ''), ('AD', ''),
+               ('RM', ret_date[0:7]), ('RD', ret_date[8:]), ('PA', '1'), ('PC', ''),
+               ('PI', ''), ('CC', ''), ('NS', ''), ('CD', '')]
+    if trip_type == 'OW':
+        payload.insert(0, ('TT', trip_type))
+    try:
+        response = requests.get('https://www.airblue.com/bookings/flight_selection.aspx',
+                                params=payload)
+        return response
+    except ConnectionError:
+        print('something wrong with your connection')
+        sys.exit()
+
+
+def get_time(path, date, kind='leaving'):
+    """a method that is finding the departure/arrive time"""
+    time_path = f'.//td[@class="time {kind}"]'
+    hours = path.xpath(f'{time_path}/text()')[0]
+    result = datetime.strptime(f'{date} {hours}', '%Y-%m-%d %I:%M %p')
+    for p in path.xpath(time_path):
+        plus_day = p.xpath('.//sup/text()')
+        if plus_day:
+            result = result + timedelta(days=1)
+    return result
+
+
+def get_duration(start, end):
+    """a method that is calculating the duration"""
+    duration = end - start
+    return duration
+
+
+def travel_deal(price, category):
+    """a method that is checking the availability of travel deal"""
+    if price == '':
+        return f'{category} travel deal is not available for the given flight.'
+    else:
+        return price
```

Comment:
you'we already checked that in l60, isn't that enough? if it's enough then this function does nothing. If it's not enough than what cases does this check cover?